geo_test.go

```diff
@@ -48,3 +49,42 @@ func BenchmarkHaversine(b *testing.B) {
 		Haversine(1.1, 2.2, 3.3, 4.4)
 	}
 }
+
+func TestDistVincenty(t *testing.T) {
```

Comment:
Since the body of the test are mostly identical except for the algorithm and threshold, could you try to refactor so that the Haversine and Vicenty methods are tested from the same test function?

I'm thinking of a structure like this:

```
cases := []struct {
  // Algorithm can be either Haversine or Vicenty
  Algorithm func(float64, float64, float64, float64) float64
  Threshold float64
} {
...
}

for c := range cases {
  // run test for the specified algorithm
}
```

Thanks

Reply:
Good idea--let me incorporate that change.